### PR TITLE
2 stranger case report

### PR DESCRIPTION
**Pull request by Mathis-Dory**
### Added

* Custom entity to give class to image
* Custom css file
* Stranger case report
* Stranger case menu
*Stranger case sockpuppet

### Changed

* Typo in archetypes

### CODE REVIEW

File: content/stranger_case/report.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+{{< imgWithClass "/images/stranger_case/tosalios.png" "Raphael Mabar" "tosalios.png" "" "stranger-case-pp" >}}
+
+- **Twitter**: [https://twitter.com/Tosalios](https://twitter.com/Tosalios)
+- **Facebook**: [https://www.facebook.com/profile.php?id=100091989057511](https://www.facebook.com/profile.php?id=100091989057511)
+- **Alias**: Tosalios
+- **Blog**: [https://tosalio.blog/](https://tosalio.blog/)
+
+His Blog has 3 snapshots on the wayback machine but nothing seems relevant except it runs WordPress, and it is associated with the username: lenainkiller0119.
+On his Twitter page we found several pieces of information: the first is that he is a fan of cycling and more particularly of the rider Thibaut PINOT. We also know that he follows the forum [https://www.jeuxvideo.com/](https://www.jeuxvideo.com/) maybe we can find something interesting there. The last big piece of information we learn is that he's a fan of the game Clash of Clans.
```
Comment (Costinteo) [inline content/stranger_case/report.md:107]: You can mention that we checked if his email is in use on jeuxvideo.com using "Forgot password" and we arrived at the conclusion that he doesn't have an account there.

File: content/stranger_case/report.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+#### Faye Terni {#ft}
+
+{{< imgWithClass "/images/stranger_case/ft.png" "Faye Terni" "ft.png" "" "stranger-case-pp" >}}
+
+- **Facebook**: [https://www.facebook.com/profile.php?id=100088501506551](https://www.facebook.com/profile.php?id=100088501506551)
+- **Linkedin**: [https://www.linkedin.com/in/faye-terni-9a7508259/](https://www.linkedin.com/in/faye-terni-9a7508259/)
+- **Email**: faye.terni@gmail.com
+
+Her older Facebook profile picture has a special instruction with hexiftool:
+FBMD0a000a700100004d1400002236000078390000853d00006449000085690000e96c0000cf7100002e7700000eae0000
```
Comment (Costinteo) [inline content/stranger_case/report.md:143]: I think this was a useless/random artifact, right? Or was it actually relevant?
Also it's "exiftool", not "hexiftool".

File: content/stranger_case/report.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+Was threatened in an email sent by
+anonymous user, they want to buy a
+property next to where she lives. Due to email, she said she changed
+phone numbers, we have personal
+email address of her, and will leave
+town. She lives at 87 Bd Laennec, Lorient, France
+
+Thanks to her email address we find out her Google calendar with the following event:
+- 24 of mai resignation from Copprethia
+- 15 & 16 may meeting Leclercq
```
Comment (Costinteo) [inline content/stranger_case/report.md:163]: Use same case and word:
"mai" -> "May"
"may" -> "May"

File: content/stranger_case/report.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+- **Professional email**: hortense.lutero@copprethia.fr
+- **Email**: lutero.hortense@gmail.com
+
+
+Was threatened in an email sent by
+anonymous user, they want to buy a
+property next to where she lives. Due to email, she said she changed
+phone numbers, we have personal
+email address of her, and will leave
+town. She lives at 87 Bd Laennec, Lorient, France
```
Comment (Costinteo) [inline content/stranger_case/report.md:159]: Put all this in a single line. Grammar could be better but it's fine. Also she lives at both 87 Bd and 85. She owns both.

**Review by Costinteo — CHANGES_REQUESTED** [review]
Sorry I'm usually very nitpick-y in reviews, lol. The report looks great though!

File: content/stranger_case/report.md (hunk 2 above)
Comment (Mathis-Dory) [reply content/stranger_case/report.md:143]: Yep can delete it there was no stegano in the chall.

File: content/stranger_case/report.md (hunk 1 above)
Comment (Mathis-Dory) [reply content/stranger_case/report.md:107]: Ok added it.

File: content/stranger_case/report.md (hunk 3 above)
Comment (Mathis-Dory) [reply content/stranger_case/report.md:163]: Resolved

File: content/stranger_case/report.md (hunk 4 above)
Comment (Mathis-Dory) [reply content/stranger_case/report.md:159]: okey done

**Comment by Mathis-Dory** [comment]
I resolved your suggestion @Costinteo check it and everything seems good merge it :)

**Pull request merged**